woocommerce/woocommerce
# Move feature flag config files to Woocommerce plugin

### All Submissions:






### Changes proposed in this Pull Request:

Feature flag generation needs to be under the Woocommerce plugin so that unit tests can be run in the wp-env environment. This PR moves the logic and adds a `build:feature-config` script to Woocommerce.

@ObliviousHarmony Can you confirm this is sufficient?

Closes https://github.com/woocommerce/woocommerce/issues/32431

### How to test the changes in this Pull Request:

1. Remove `plugins/woocommerce/includes/react-admin/feature-config.php` if its in your file system. This file will be generated by script.
2. Run `nx build woocommerce` and confirm the `feature-config.php` file has reappeared.

### Other information:




### FOR PR REVIEWER ONLY:
